demo-ui/app/uk/gov/ons/addressIndex/demoui/views/index.scala.html

```diff
@@ -0,0 +1,33 @@
+@()(implicit messages: Messages)
```

Comment:
This still works? We moved MessagesApi from the implicit parameter to the explicit one in the Controller.